EIPS/eip-2400.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 2400
+title: URL Format for Transaction Receipts
+author: Ricardo Guilherme Schmidt (@3esmit), Eric Dvorsak (@yenda)
+discussions-to: https://ethereum-magicians.org/t/eip-2400-transaction-receipt-uri/
+type: Standards Track
+category: ERC
+status: Review
+created: 2019-11-05
+requires: 155, 831
+---
+
+## Simple Summary
+
+A standardized URI for transaction receipt with complete information about transaction.
+
+A standard way of representing a submitted transaction.
+
+## Abstract
+
+A transaction hash is not very meaningful on its own, because it looks just like any other hash, and it might lack important information for reading a transaction. 
+This standard includes all needed information for displaying a transaction and it's details, such as `chainId`, `method` signature called and `events` signatures emitted.
+
+### Use-cases 
+
+Transaction Receipt URIs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URI format allows for instant invocation of the user’s preferred transaction explorer application. Such as:
+
+- In web3 (dapps, mining pools, exchanges), links would automatically open user's preferred transaction explorer; 
+- In wallets, for users sharing transaction receipts easier; 
+- In chat applications, as a reply to an [ERC-681] transaction request;
+- In crypto vending machines, a QRCode can be displayed when transactions are submitted;
+- Anywhere transaction receipts are presented to users.
+
+## Motivation
+
+Interoperability for web3: wallets, browsers and messengers.
+
+## Specification
+
+### Syntax
+
+Transaction receipt URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    receipt                 = erc831_part transaction_hash [ "@" chain_id ] [ "?" parameters ]
+    erc831_part             = "ethereum:tx-" 
```

Comment:
EIP-831 is stagnant and should not be referenced.

I suggest choosing a different schema. Perhaps `ethereum-tx`?

Reply:
Perhaps just dont mention 831 and keep it as it is.

Reply:
If you want, you could submit your own PR to move it out of stagnant (and add yourself as an author), and if ligi doesn't respond, either micah or lightclient could override the author approval requirement.